.github/workflows/ci.yml

```
@@ -187,7 +187,7 @@ jobs:
       uses: actions/checkout@v3
 
     - name: Run GoReleaser
-      uses: goreleaser/goreleaser-action@v3
+      uses: goreleaser/goreleaser-action@v4
       with:
         version: latest
         args: release --rm-dist
```

```suggestion
        args: release --clean
```
Can you update this as well? It got deprecated.